src/pages.ts

```diff
@@ -0,0 +1,2 @@
+export * as PageTypes from './lib/pages/interfaces';
```

Comment:
What do you think about namespacing the types?

I think this approach can get a bit verbose

Reply:
the types are already namespaced under `@charmverse/core/pages`